src/lifecycle/application.rs

```diff
@@ -37,26 +37,53 @@ impl<T: State> Application<T> {
             self.state.event(&self.event_buffer[i], &mut self.window)?;
         }
         self.event_buffer.clear();
+
         let current = current_time();
         self.accumulator += current - self.last_update;
         self.last_update = current;
         let mut ticks = 0;
-        while self.accumulator > 0.0 && (self.window.max_ticks() == 0 || ticks < self.window.max_ticks()) {
-            self.state.update(&mut self.window)?;
+        let update_rate = self.window.update_rate();
+        while self.accumulator > 0.0 && (self.window.max_updates() == 0 || ticks < self.window.max_updates()) {
+            self.state.update(&mut self.window, update_rate)?;
```

Comment:
Is there still a reason to pass in dt if it's just `window.update_rate()`?

Reply:
Probably not. I guess that can go then?

Reply:
Oh nevermind, I guess using
- [Breaking] Remove `WindowBuilder` and create a new `Settings` struct to replace it
is enough. Or do you want vsync to have a separate entry?

Reply:
@Byter09 I think vsync probably warrants its own entry in the changelog
@999eagle I would say probably not pass in DT, because as of right not it would just be duplicating `window.update_rate()` at present.